src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala

```diff
@@ -105,12 +106,13 @@ class ConfigSessionProvider(system: ActorSystem, config: Config) extends Session
     }
 
```

Comment:
let's add a
```
val sessionSettings: CassandraSessionSettings = CassandraSessionSettings(config)
```

here instead, then we use the same config parsing as in other place.
Don't catch exception because if the config is wrong it should be visible.

Reply:
updated per your suggestions